app/javascript/packages/components/troubleshooting-options.tsx

```diff
@@ -35,7 +35,7 @@ function TroubleshootingOptions({
   return (
     <section className="troubleshooting-options">
       <HeadingTag className="troubleshooting-options__heading">
-        {heading ?? t('components.troubleshooting_options.default_heading')}
+        {heading ?? t('components.troubleshooting_options.ipp_heading')}
```

Comment:
This is the base TroubleshootingOptions React component, which is meant to be generally reusable. With my previous comment I meant that we can render it with the `heading` prop that's specific to where it's used:

https://github.com/18F/identity-idp/blob/3c69b3166266d3aba5df3cc208fbb9b98c948761/app/javascript/packages/document-capture/components/in-person-troubleshooting-options.tsx#L29-L30